crates/cac_client/src/interface.rs

```diff
@@ -105,7 +109,7 @@ pub extern "C" fn cac_new_client(
 
     // println!("Creating cac client thread for tenant {tenant}");
     let local = task::LocalSet::new();
-    local.block_on(&Runtime::new().unwrap(), async move {
+    local.block_on(&CAC_RUNTIME, async move {
```

Comment:
We can do this for all functions expect `cac_start_polling_update` since I am expecting the FFI language to spawn a thread to run this function in the background